FirebaseFunctionsSwift/Sources/FunctionsContext.swift

```diff
@@ -38,21 +45,21 @@ internal class FunctionsContextProvider: NSObject {
 //
 //  }
 
-  internal func getContext(_ completion:  @escaping ((FunctionsContext, Error?) -> Void)) {
+  internal func getContext(_ completion: @escaping ((FunctionsContext, Error?) -> Void)) {
     let dispatchGroup = DispatchGroup()
 
-    var authToken: String? = nil
-    var appCheckToken: String? = nil
-    var error: Error? = nil
+    var authToken: String?
+    var appCheckToken: String?
+    var error: Error?
```

Comment:
Is this a swiftlint preference? I kind of like having the `nil` explicitly declared there, although maybe that's my ObjC days talking. WDYT?

Reply:
Yes this is from SwiftLint.  As a Swift newbie, I'm generally fine to go with SwiftLint recommendations and learn than the nil is implied - but open to having a different opinion for our code.